keysManagement/keysHolder.go

```diff
@@ -63,10 +69,40 @@ func checkVirtualPeersHolderArgs(args ArgsVirtualPeersHolder) error {
 		return fmt.Errorf("%w for MaxRoundsWithoutReceivedMessages, minimum %d, got %d",
 			errInvalidValue, minRoundsWithoutReceivedMessages, args.MaxRoundsWithoutReceivedMessages)
 	}
+	if len(args.NamedIdentities) == 0 {
+		return errMissingNamedIdentity
```

Comment:
we should not consider this a critical error. Not even a log.Debug is required. If we did not provide some named identities, use the default ones on all managed keys.